operator/gitops/argocd/grafana/dashboards/pipeline-service-dashboard.json

```diff
@@ -91,10 +36,7 @@
       "type": "row"
     },
     {
-      "datasource": {
-        "type": "prometheus",
-        "uid": "${DS_PROMETHEUS-APPSTUDIO-DS}"
-      },
+      "datasource": {},
```

Comment:
Is the static check perhaps complaining about this @Roming22 ?

FWIW I just copied panels and did not change datasources ... I think this grafana export function is flaky

Reply:
I'm going to undo this manually, see if the panels render in my dev_setup deployment